<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestMainWindow</class>
 <widget class="QWidget" name="TestMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestMainWindow</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/star.png</normaloff>:/icon/star.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>toolButton</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/star.png</normaloff>:/icon/star.png</iconset>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
   <property name="arrowType">
    <enum>Qt::NoArrow</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>109</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>109</width>
     <height>70</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QRadioButton" name="radioButton_3">
      <property name="styleSheet">
       <string notr="true">QRadioButton{
	font-size:14px;
	color:white;
	background:qlineargradient(x1:0.5, y1:0, x2:0.5, y2:1,stop:0 		rgb(55,170,42), stop: 1 rgb(0,141,0));
	/*border: 1px solid rgb(42,161,31);*/
	border-radius:5px;  
	font: 75 12pt &quot;Microsoft YaHei UI&quot;;


}
QRadioButton:hover{
	font-size:14px;
	color:white;
	background:qlineargradient(x1:0.5, y1:0, x2:0.5, y2:1,stop:0 rgb(100,200,50), stop: 1 rgb(11,141,65));
	border: 1px solid rgb(42,161,31);
	border-radius:5px;  
	font: 75 12pt &quot;Microsoft YaHei UI&quot;;

}
QRadioButton:checked{
color:blue;
}

QRadioButton::indicator {
   width: 10px; /*根据你想要的距离，自己调整吧*/
   height: 0px;
}</string>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioButton_4">
      <property name="styleSheet">
       <string notr="true">QRadioButton{
	font-size:14px;
	color:white;
	background:qlineargradient(x1:0.5, y1:0, x2:0.5, y2:1,stop:0 		rgb(55,170,42), stop: 1 rgb(0,141,0));
	/*border: 1px solid rgb(42,161,31);*/
	border-radius:5px;  
	font: 75 12pt &quot;Microsoft YaHei UI&quot;;


}
QRadioButton:hover{
	font-size:14px;
	color:white;
	background:qlineargradient(x1:0.5, y1:0, x2:0.5, y2:1,stop:0 rgb(100,200,50), stop: 1 rgb(11,141,65));
	border: 1px solid rgb(42,161,31);
	border-radius:5px;  
	font: 75 12pt &quot;Microsoft YaHei UI&quot;;

}
QRadioButton:checked{
color:blue;
}
QRadioButton::indicator {
   width: 10px; /*根据你想要的距离，自己调整吧*/
   height: 0px;
}
</string>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>131</width>
     <height>92</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>a</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>b</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string>c</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>listWidgetadd</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>列1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>列2</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>1</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>520</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>treeWidgeAdd</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>新建行1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>列1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>列2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>1</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>360</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>更改tableWidget</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>460</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>69</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Page 1</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>69</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Page 2</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>620</y>
     <width>211</width>
     <height>80</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>37</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>172</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>CommandLinkButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>页</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>740</y>
     <width>120</width>
     <height>81</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QCommandLinkButton" name="commandLinkButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>172</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>CommandLinkButton</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>740</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>变更StackedWidget</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelPixMap">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>680</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="movieLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>640</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
